### ISSUE TITLE
pullToRefresh don't work when RadListView is nested into a RadSideDrawer

### ISSUE THREAD

**Issue by leocaseiro** [telerik/nativescript-ui-samples-angular#11]
Hi, I'm trying to implement a RadListView which is nested into a RadSideDrawer, however, the pullToRefresh don't work.

If I remove the RadSideDrawer, the pullToRefresh works perfectly, so I believe there's a conflict.

Bellow I'm posting my html for testing:

```html
<RadSideDrawer #drawer >
    <template drawerSide>
        <StackLayout class="sideStackLayout">
            <StackLayout class="sideTitleStackLayout">
                <Label text="Navigation Menu"></Label>
            </StackLayout>
            <StackLayout class="sideStackLayout">
                <Label text="Primary" class="sideLabel sideLightGrayLabel"></Label>
                <Label text="Social" class="sideLabel"></Label>
                <Label text="Promotions" class="sideLabel"></Label>
                <Label text="Labels" class="sideLabel sideLightGrayLabel"></Label>
                <Label text="Important" class="sideLabel"></Label>
                <Label text="Starred" class="sideLabel"></Label>
                <Label text="Sent Mail" class="sideLabel"></Label>
                <Label text="Drafts" class="sideLabel"></Label>
            </StackLayout>
        </StackLayout>
    </template>
    <template drawerMain>
        <GridLayout orientation="vertical" rows="auto, *">
            <RadListView row="1" [items]="dataItems" pullToRefresh="true" (pullToRefreshInitiated)="onPullToRefreshInitiated($event)" [listViewLayout]="layout">
                <template listItemTemplate let-item="item">
                    <StackLayout class="itemTemplateStackLayout" orientation="vertical">
                        <StackLayout class="innerOtemTemplateStackLayout" orientation="horizontal">
                            <Image [src]="item.image" stretch="aspectFit" height="100" width="100"></Image>
                            <StackLayout class="labelsStackLayout" orientation="vertical">
                                <Label class="labelName" [text]="item.name"></Label>
                                <Label class="labelTitle" [text]="item.title" textWrap="true"></Label>
                                <Label class="labelText" [text]="item.text" textWrap="true"></Label>
                            </StackLayout>
                        </StackLayout>
                    </StackLayout>
                </template>
                <template listViewHeader>
                    <GridLayout backgroundColor="#65a565">
                        <Label text="This is the list header"></Label>
                        <Button (tap)="toggleMenu()" text="Open Menu"></Button>
                    </GridLayout>
                </template>
            </RadListView>
        </GridLayout>
   </template>
</RadSideDrawer>
```

My Component code is:

```javascript
export class ListComponent implements OnInit {
    private _dataItems: ObservableArray<DataItem>;
    private _numberOfAddedItems;
    private _layout: listViewModule.ListViewLinearLayout;

    @ViewChild(RadSideDrawerComponent) public drawerComponent: RadSideDrawerComponent;
    private drawer: SideDrawerType;

    constructor(private _changeDetectionRef: ChangeDetectorRef) {
    }

    ngOnInit() {
        this.layout = new listViewModule.ListViewLinearLayout();
        this.layout.scrollDirection = "Vertical";
        this.initDataItems();
        this._changeDetectionRef.detectChanges();
    }

    ngAfterViewInit() {
        this.drawer = this.drawerComponent.sideDrawer;
    }

    public get dataItems(): ObservableArray<DataItem> {
        return this._dataItems;
    }

    public get layout(): listViewModule.ListViewLinearLayout {
        return this._layout;
    }

    public set layout(value: listViewModule.ListViewLinearLayout) {
        this._layout = value;
    }

    public onPullToRefreshInitiated(args: listViewModule.ListViewEventData) {
        var that = new WeakRef(this);
        Timer.setTimeout(function () {
            var initialNumberOfItems = that.get()._numberOfAddedItems;
            for (var i = that.get()._numberOfAddedItems; i < initialNumberOfItems + 2; i++) {
                if (i > posts.names.length - 1) {
                    break;
                }
                var imageUri = Application.android ? posts.images[i].toLowerCase() : posts.images[i];

                that.get()._dataItems.splice(0, 0, new DataItem(i, posts.names[i], "This is item description", posts.titles[i], posts.text[i], "res://" + imageUri));
                that.get()._numberOfAddedItems++;
            }
            var listView = args.object;
            listView.notifyPullToRefreshFinished();
        }, 1000);
    }

    private initDataItems() {
        this._dataItems = new ObservableArray<DataItem>();
        this._numberOfAddedItems = 0;
        for (var i = 0; i < posts.names.length - 15; i++) {
            this._numberOfAddedItems++;
            if (Application.android) {
                this._dataItems.push(new DataItem(i, posts.names[i], "This is item description", posts.titles[i], posts.text[i], "res://" + posts.images[i].toLowerCase()));
            }
            else {
                this._dataItems.push(new DataItem(i, posts.names[i], "This is item description", posts.titles[i], posts.text[i], "res://" + posts.images[i]));
            }
        }
    }

    public toggleMenu() {
        this.drawer.toggleDrawerState();
    }
}
```

**Comment by VladimirAmiorkov** [comment]
hi @leocaseiro ,

This was a know issue which has been resolved in the latest **1.3.1** version of [nativescript-telerik-ui](https://www.npmjs.com/package/nativescript-telerik-ui). Please give it a try.

**Comment by leocaseiro** [comment]
Hi @VladimirAmiorkov. Thank you for your reply. I'll make tests and I'll let you know if I have any issues.
